delete/wp-site

```diff
@@ -9,8 +9,11 @@ fi
 
 cd "$(dirname $0)"
 # Read config file with paths to WP-installs and usernames
-source ../conf
+source /usr/local/default/etc/wp-mass-admin/wpma.conf
 
+if [[ -e /usr/local/etc/wp-mass-admin/wpma.conf ]]; then
+    source /usr/local/etc/wp-mass-admin/wpma.conf
+fi
```

Comment:
Hmm, it's not a clear cut case now that I read the code again. I'd have to ask some questions first:

1. Is there anything the default conf in a clean installation?
2. Which conf file is automatically written to?
3. IFF the automatic addition of site names is to the non-default-conf, does it make sense to attempt to delete the given site since it's reasonable to assume that there are none?